### ISSUE TITLE
Sent messages are rendered as markdown

### ISSUE THREAD

**Issue by The-Compiler** [darrenburns/elia#25]
On the 0.5.0 release,

![image](https://github.com/darrenburns/elia/assets/625793/b89f2e06-73bf-4906-96ac-5c748cead72d)

renders as:

![image](https://github.com/darrenburns/elia/assets/625793/e26f3dda-8412-4caf-b404-4406b79e2fbb)

And more generally, markdown is rendered in what the user types. However, ChatGPT does not interpret input as Markdown as far as I know, so it might make sense to display messages coming from the user as plain text entirely?

**Comment by kjrstory** [comment]
It seems simple to fix this. If you change the part of the chatbox.py file as below
```python
...
from rich.text import Text
...
    @property
    def markdown(self) -> Markdown:
        return Markdown(self.message.content or "")

    @property
    def text(self) -> Text:
        return Text(self.message.content or "")

    def render(self) -> RenderableType:
        if self.message.type == "human":
            return self.text
        else:
            return self.markdown
...
```
There are three types of messages: human, ai, and system.
I confirmed that the input was rendered as text and the output was rendered as markdown.
![](https://github.com/darrenburns/elia/assets/7046937/453aaaf5-c2d6-42ec-8ce7-3534480bcd04)

However, I cannot judge whether it makes sense. The code may become complicated, but it will also be possible to choose whether to render the user's input as markdown or text.

**Comment by darrenburns** [comment]
This has been fixed in 1.0.0
